Docker/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:22.04
+
+RUN apt-get update
+RUN apt-get install wget python3 zlib1g-dev git build-essential -y
```

Comment:
Nit: put in alphabetical order. More importantly, you may want to pin to particular versions.

Reply:
I wouldn't try to pin these to specific versions and it should be unnecessary if you are building to a specific OS version.